.github/workflows/translations-download-workflow.yml

```diff
@@ -0,0 +1,34 @@
+name: New Crowdin translations PR
+
+on:
+  workflow_dispatch:
```

Comment:
When triggered, this workflow creates a PR with the current approved translations in Crowdin. 

Unfortunately I wasn't able to add the automatic workflow we wanted to set up
(New translations pushed to dev -> Crowdin automatically translates -> A PR is created automatically)
because triggering the automatic translations feature is behind a paywall, so I wasn't able to even test it:

![image](https://github.com/user-attachments/assets/2fff466d-fef6-4f4f-acc8-c0cd91ef78dd)

![image](https://github.com/user-attachments/assets/e585282f-6f8f-4092-a58d-3337a9623c09)


Minimum plan to enable it would be "Team".
![image](https://github.com/user-attachments/assets/d7dde389-d58e-4811-a95a-b93a4fd17172)

Same for webhooks:
![image](https://github.com/user-attachments/assets/ea6d2dd4-4024-469b-b43e-4c8ad0d2a0a4)